kasa/cli.py

```diff
@@ -339,8 +340,9 @@ def _nop_echo(*args, **kwargs):
             credentials=credentials,
         )
 
-    # Skip update for wifi & raw-command, and if factory was used to connect
-    if ctx.invoked_subcommand not in ["wifi", "raw-command"] and not device_family:
+    # Skip update for wifi & raw command, and if factory was used to connect
+    SKIP_UPDATE_COMMANDS = ["wifi", "raw-command", "command"]
```

Comment:
I'd probably define the constant in the outer scope so it doesn't call `PyList_New` every time

Reply:
Doesn't really matter for the cli usage, but I pulled it up.